webrtc/videoroom/lib/videoroom/pipeline.ex

```diff
@@ -222,17 +226,33 @@ defmodule VideoRoom.Pipeline do
     {{:ok, spec: spec}, state}
   end
 
-  def handle_notification({:vad, val}, {:endpoint, endpoint_id}, _ctx, state) do
-    display_engine = state.display_engine
-    {actions, display_engine} = DisplayEngine.vad_notification(display_engine, val, endpoint_id)
-    {{:ok, actions}, %{state | display_engine: display_engine}}
+  def handle_notification(
+        {:signal, {:sdp_offer, _} = message},
+        {:endpoint, peer_pid},
+        _ctx,
+        state
+      ) do
+    participants =
+      state.endpoints
+      |> Enum.map(fn {_, %Endpoint{inbound_tracks: tracks, ctx: ctx}} ->
+        %{display_name: ctx.display_name, mids: Map.keys(tracks)}
+      end)
+
+    send(peer_pid, {:signal, message, participants})
+    {:ok, state}
   end
 
   def handle_notification({:signal, message}, {:endpoint, peer_pid}, _ctx, state) do
     send(peer_pid, {:signal, message})
     {:ok, state}
   end
 
+  def handle_notification({:vad, val}, {:endpoint, endpoint_id}, _ctx, state) do
+    display_engine = state.display_engine
+    {actions, display_engine} = DisplayEngine.vad_notification(display_engine, val, endpoint_id)
+    {{:ok, actions}, %{state | display_engine: display_engine}}
+  end
```

Comment:
This has been already added on master. Is it a fix after reabse? :thinking:

Reply:
Merge probably moved some lines in a way that this looks like something has been inserted.